base_export_manager/models/res_users.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 - Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from openerp import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.v7
+    def get_export_models(self, cr, uid):
+        return self.fetch_export_models(cr, uid)
+
+    @api.v8
```

Comment:
Couldn't we combine the two api.7/8 methods into an api.multi instead? 

IMO the only reason to fracture the APIs is if they are already fractured in downstream code. This method is new, so I don't see any reason it can't fully be the new api.